src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ThrowHelper.cs

```diff
@@ -61,17 +61,17 @@ internal static Exception GetExceptionForMsQuicStatus(int status, string? messag
 
         return ex;
 
-        static Exception GetExceptionInternal(int status, string? message)
+        static Exception GetExceptionInternal(int status, ulong? errorCode, string? message)
```

Comment:
```suggestion
        static Exception GetExceptionInternal(int status, long? errorCode, string? message)
```
Than the casts won't be necessary.